app/controllers/editor/categories_controller.rb

```diff
@@ -43,6 +43,24 @@ def destroy
     redirect_to editor_categories_url, notice: "カテゴリが削除されました"
   end
 
+  # move to left
+  def move_higher
+    category = categories.find(params[:id])
+    target_category = category.higher_items.last
+    category.move_to_left_of(target_category) if target_category
```

Comment:
When `target_category` is `nil`?
If it is bad case, the error should be raised?